rubocop.minitest.yml

```diff
@@ -0,0 +1,2 @@
+Minitest:
+  Enabled: true
```

Comment:
I haven't bothered filling out most of these files, but presumably they'd look like:

```yaml
PluginName:
  Enabled: true

# ...config for cops where we want to diverge from the default...
```

Reply:
We could alternatively call these after the gem names (e.g. `rubocop-minitest.yml`). I just went with what we use in Core.

Reply:
I prefer to not use defaults. Let's always curate cops.